test/webserv/integration/test_http_delete.py

```
@@ -0,0 +1,171 @@
+import os
+import time
+
+import pytest
+from common_functions import read_file, send_request_and_assert_response
```

Ruff

`common_functions.read_file` imported but unused

Remove unused import: `common_functions.read_file`

(F401)